### ISSUE TITLE
[1.18.2] BitStorage による液体の重複

### ISSUE THREAD

**Issue by karikusa** [ChiselsAndBits/Chisels-and-Bits#1071]
Version: 1.18.2 Forge 40.2.0
C&B: 115 and 116
Expected behavior: Right-clicking BitStorage with a bucket of liquid fills it with liquid and returns an empty bucket.
What it actually does: Retains a bucket of liquid when filling BitStorage with liquid.
In this state, if you move it from BitStorage to BitBag, you can't fill it with the remaining liquid bucket, but if you install it in the world and collect it again, you can fill it in BitStorage again.
In other words, if you have a bucket of liquid, you can increase the liquid Bit.
